aws/required-permissions.md

```diff
@@ -129,3 +131,82 @@ Check out the templates directly for more information.
+| `s3:GetObject` | Action | Retrieve the object to be scanned or quarantined from Amazon S3. |
+| `s3:GetObjectTagging` | Action | List the tags of the scanned object. |
+| `s3:PutObject` | Action | Put the malicious object to quarantine bucket. |
+| `s3:PutObjectTagging` | Action | Put tags on the scanned object. |
+
+### Permissions required by ManagementRole
+
+| Permission | Type | Description |
+| --- | --- | --- |
+| `cloudformation:CreateChangeSet` | Action | Creates a list of changes that will be applied to a stack so that you can review the changes before executing them. |
+| `cloudformation:DescribeStacks` | Action | Allow Cloud One File Storage Security to describe the managed account scanner stack. |
+| `cloudformation:DescribeStackEvents` | Action | Allow Cloud One File Storage Security to describe the events in the managed account scanner stack. |
+| `cloudformation:DescribeStackResource` | Action | Allow Cloud One File Storage Security to describe the deployed resources in the managed account scanner stack. |
+| `cloudformation:DescribeStackResources` | Action | Allow Cloud One File Storage Security to describe the deployed resources in the managed account scanner stack. |
+| `cloudformation:DetectStackDrift` | Action | Allow Cloud One File Storage Security to detect the drift of the managed account scanner stack. |
+| `cloudformation:DetectStackResourceDrift` | Action | Allow Cloud One File Storage Security to detect the drift of the deployed resources in the managed account scanner stack. |
+| `cloudformation:DescribeStackResourceDrifts` | Action | Allow Cloud One File Storage Security to describe the drift of the deployed resources in the managed account scanner stack. |
+| `cloudformation:GetStackPolicy` | Action | Allow Cloud One File Storage Security to get the managed account scanner stack policy. |
+| `cloudformation:GetTemplate` | Action | Allow Cloud One File Storage Security to get the template of the managed account scanner stack. |
+| `cloudformation:ListStackResources` | Action | Allow Cloud One File Storage Security to list the deployed resources in the managed account scanner stack. |
+| `iam:GetRolePolicy` | Action | Allow Cloud One File Storage Security to get the management role policies. |
+| `lambda:CreateAlias` | Action | Allow Cloud One File Storage Security to create an alias for Lambda functions in the managed account scanner stack. |
+| `lambda:DeleteAlias` | Action | Allow Cloud One File Storage Security to delete the alias of Lambda functions in the managed account scanner stack. |
+| `lambda:GetAlias` | Action | Allow Cloud One File Storage Security to describe the alias of Lambda functions in the managed account scanner stack. |
+| `lambda:GetEventSourceMapping` | Action | Allow Cloud One File Storage Security to describe the event source mapping of Lambda functions in the managed account scanner stack. |
```

Comment:
```suggestion
| `lambda:GetEventSourceMapping` | Action | Allows Cloud One File Storage Security to describe the event source mapping of Lambda functions in the managed account scanner stack |
```
The third cell in this row contains a clause phrase, not a full sentence, so doesn't require a period. The verb form should be the same for each clause in this column. Because the implied subject (the managed policy) is singular, the verb should be singular which typically ends in "s".